src/aura/STG_CMP_System/STG_CMP_System.cmp

```diff
@@ -200,7 +200,9 @@
         </div>
         
         <div class="slds-col slds-size--1-of-2" >
-            <c:CMP_Acc_Naming_Dropdown isView="{!v.isView}"
+            <c:CMP_Acc_Naming_Dropdown 
+                class="admin-account-naming"
```

Comment:
This is similar to how we include the other CMP_* lightning components in the HEDA Settings.

Without defining this, then there is no need to even have the class attribute in the CMP_Acc_Naming_Dropdown component. I just picked a random name here - it doesn't look like this is used in any of our robot tests yet as the robot tests passed successfully.

Reply:
@robertbwatson I think this works. I'm trying to figure out though where the "requirement" for having to have the attribute "class" specified. I see many of our components have it, but don't know where/why it's being forced/accessed.

Reply:
@robertbwatson Never mind...I'm blind and seeing it now. It's because every one of those elements in that component are referencing v.class, but we didn't have an attribute there. 

I'm good with this.

Reply:
@lifewithryan Yeah it's not "required" but without a value specified, the class ends up being something like "-output-text" instead of "admin-account-naming-output-text"